CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## master
 
+## [0.4] - 2020-12-22
+- [added] Multiple lexicons can be save and stored (source and destination languages are unique together)
+- [changed] Now there can not be a duplicated word in a lexicon. (unique together lexicon and term)
+- [changed] Avoid repeated entries. (unique together word, translation and variation) 
+- [changed] Importation require to be passed a lexicon_code of an already existing lexicon.
+- [changed] API: View WordViewSet requires a parameter l (lexicon_code) to perform search and near methods.
+- [added] Admin panel and an url_admin_panel in Word model to access it directly.
```

Comment:
Typo? Admin panel **and** an url_admin_panel in Word model to access it directly.

Reply:
Rewritten to be more accurate